server/lib/s3backup/__init__.py

```diff
@@ -22,6 +22,13 @@ class Status(Enum):
     SUCCESS = 0
     FAIL = 1
 
+class NoSuchKey(Exception):
+    pass
+
+# a generic exception for testing - only raised by the mocked get_object()
+# so that get_tarball_header() can be tested.
+class Get_Object_Exception(Exception):
```

Comment:
This exception appears to be only used by the "mock" code, can you consider renaming it to `MockGetObjectException`?  And also move the comment as a doc string?